js/message_test.js

```diff
@@ -1052,13 +1052,4 @@ describe('Message test suite', function() {
     assertNan(message.getDefaultDoubleField());
   });
 
-  it('testSerializePreservesEmptyNestedField', function() {
```

Comment:
Why is this test removed?

Reply:
This one tries to use the internal-only JSPB wire format.